.gitignore

```diff
@@ -20,3 +20,5 @@ ruff.toml
 /keymaps.db
 /.vscode/
 /reports/
+/config.yml
+/manifest.csv
```

Comment:
Are these two ignores local and you forgot about them before committing or is there a specific reason for them here? Could add the context to PR description.

Reply:
I forgot to add them to the description, updated.